Gameplay/generated/SaveAndLoadMethods.cpp

```diff
@@ -576,6 +587,16 @@ void Hachiko::Scripting::PlayerController::OnLoad()
 		_max_dash_charges = load_node["'_max_dash_charges@unsigned'"].as<unsigned>();
 	}
 
+	if (load_node["'_dash_trail@GameObject*'"].IsDefined())
+	{
+		_dash_trail = SceneManagement::FindInCurrentScene(load_node["'_dash_trail@GameObject*'"].as<unsigned long long>());
```

Comment:
you could use UID instead of unsigned long long

Reply:
that's some auto generated stuff